src/xbundle.py

```diff
@@ -410,13 +413,13 @@ def write_xml(x):
         #print elem
         if elem.tag=='descriptor':
             # print "--> %s" % list(elem)
-            self.export_xml_to_directory(elem[0])	# recurse on children, depth first
+            self.export_xml_to_directory(elem[0])   # recurse on children, depth first
             elem.tag = elem.get('tag')
             elem.set('url_name',elem.get('url_name'))
             elem.attrib.pop('tag')
-            # self.export_xml_to_directory(elem)	# recurse on this tag
+            # self.export_xml_to_directory(elem)    # recurse on this tag
 
-        elif elem.tag==etree.Comment:			# comment <!-- foo -->
+        elif elem.tag==etree.Comment:           # comment <!-- foo -->
```

Comment:
Are those whitespaces added by an script? @ericfab179

Reply:
@morenol Yes I think modernize added it